webapp/styles/TopNav.module.scss

```diff
@@ -0,0 +1,90 @@
+.TopNav {
+	padding: 1vh;
```

Comment:
if we decide to go with `vh`, `rem`, or `px`. We should be consistent throughout. Usually, you need to take a look around before you code to see what unit is used in the system already